jena-permissions/src/main/java/org/apache/jena/permissions/SecurityEvaluator.java

```diff
@@ -213,7 +210,7 @@
 	 * @throws AuthenticationRequiredException
 	 *             if user is not authenticated and is required to be.
 	 */
-	public boolean evaluate(Object principal, Action action, Node graphIRI)
+	boolean evaluate(Object principal, Action action, Node graphIRI)
```

Comment:
Minor: "public" is used elsewhere on interface methods.

Reply:
fixed